docs/reference/search/request-body.asciidoc

```diff
@@ -37,106 +36,10 @@ The search request can be executed with a search DSL, which includes the
 
 include::{docdir}/rest-api/common-parms.asciidoc[tag=index]
 
-
 [[search-request-body-api-request-body]]
 ==== {api-request-body-title}
 
-`allow_partial_search_results`:: 
-  (Optional, boolean) Set to `false` to fail the request if only partial results 
-  are available. Defaults to `true`, which returns partial results in the event 
-  of timeouts or partial failures You can override the default behavior for all 
-  requests by setting `search.default_allow_partial_results` to `false` in the 
-  cluster settings.
-
-`batched_reduce_size`:: 
-  (Optional, integer) The number of shard results that should be reduced at once 
-  on the coordinating node. This value should be used as a protection mechanism 
-  to reduce the memory overhead per search request if the potential number of 
-  shards in the request can be large.
-
-[[ccs-minimize-roundtrips]]
-`ccs_minimize_roundtrips`::
-  (Optional, boolean) If `true`, the network round-trips between the 
-  coordinating node and the remote clusters ewill be minimized when executing 
-  {ccs} requests. See <<ccs-network-delays>>. Defaults to `true`. 
-
-include::{docdir}/rest-api/common-parms.asciidoc[tag=from]
-
-`request_cache`::
-  (Optional, boolean) If `true`, the caching of search results is enabled for 
-  requests where `size` is `0`. See <<shard-request-cache>>.
-
-include::{docdir}/rest-api/common-parms.asciidoc[tag=search_type]
-
-`size`:: 
-  (Optional, integer) The number of hits to return. Defaults to `10`.
-
-include::{docdir}/rest-api/common-parms.asciidoc[tag=terminate_after]
-
-include::{docdir}/rest-api/common-parms.asciidoc[tag=search_timeout]
-
-
-Out of the above, the `search_type`, `request_cache` and the 
-`allow_partial_search_results` settings must be passed as query-string 
```

Comment:
does this in reverse mean the `batched_reduce_size` and `ccs_minimize_roundtrips` should also go to the "body" section on the target page? Currently they seem to be only listed under Query Parameters. I don't know if they are allowed in both places tbh but we can check...

Reply:
I tested this by running off the `master` branch and it appears that neither `batched_reduce_size` nor `ccs_minimize_roundtrips` are valid request body parameters. Both work fine as query parameters though.

### Setup
Index a simple 
```
PUT /my_index/_doc/0
{
  "user": "kimchy"
}
```

### `ccs_minimize_roundtrips`
```
GET /my_index/_search
{
    "ccs_minimize_roundtrips": false,
    "query" : {
        "term" : { "user" : "kimchy" }
    }
}
```
Response:

```
{
  "error" : {
    "root_cause" : [
      {
        "type" : "parsing_exception",
        "reason" : "Unknown key for a VALUE_BOOLEAN in [ccs_minimize_roundtrips].",
        "line" : 3,
        "col" : 32
      }
    ],
    "type" : "parsing_exception",
    "reason" : "Unknown key for a VALUE_BOOLEAN in [ccs_minimize_roundtrips].",
    "line" : 3,
    "col" : 32
  },
  "status" : 400
}
```

### `batched_reduce_size`
```
GET /my_index/_search
{
    "batched_reduce_size": 512,
    "query" : {
        "term" : { "user" : "kimchy" }
    }
}
```

Response:

```
{
  "error" : {
    "root_cause" : [
      {
        "type" : "parsing_exception",
        "reason" : "Unknown key for a VALUE_NUMBER in [batched_reduce_size].",
        "line" : 3,
        "col" : 28
      }
    ],
    "type" : "parsing_exception",
    "reason" : "Unknown key for a VALUE_NUMBER in [batched_reduce_size].",
    "line" : 3,
    "col" : 28
  },
  "status" : 400
}
```

Reply:
Thanks for checking.